exchanges/kline/types.go

```diff
@@ -0,0 +1,50 @@
+package kline
+
+import (
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+)
+
+// Consts here define basic time intervals
+const (
+	OneMin     = time.Minute
+	ThreeMin   = 3 * time.Minute
+	FiveMin    = 5 * time.Minute
+	FifteenMin = 15 * time.Minute
+	ThirtyMin  = 30 * time.Minute
+	OneHour    = 1 * time.Hour
+	TwoHour    = 2 * time.Hour
+	FourHour   = 4 * time.Hour
+	SixHour    = 6 * time.Hour
+	TwelveHour = 12 * time.Hour
+	OneDay     = time.Hour * 24
+	ThreeDay   = time.Hour * 72
+	OneWeek    = time.Hour * 168
+)
+
+// Item holds all the relavent information for internal kline elements
+type Item struct {
+	Exchange string
+	Pair     currency.Pair
+	Asset    asset.Item
+	Interval time.Duration
+	Candles  []Candle
+}
+
+// Candle holds historic rate information.
+type Candle struct {
+	Time   time.Time
+	Open   float64
+	High   float64
+	Low    float64
+	Close  float64
+	Volume float64
+}
+
+// HeartBeat denotes open and close times on a chart respective to time period
+type HeartBeat struct {
+	Open  time.Time
```

Comment:
Could these names change? It throws me that there are multiple open and closes being assigned in the same area with different types